gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -17,4 +17,12 @@ open class AnvilExtension {
    * By default this feature is disabled.
    */
   var generateDaggerFactories = false
+
+  /**
+   * There are occasions where consumers of Anvil are only interested in generating Dagger
+   * factories to speed up build times and don't want to make use of the other features. With this
+   * flag Anvil will only generate Dagger factories and skip all other steps. If this flag is set
```

Comment:
If it's a requirement that generateDaggerFactories has the same value, would it be possible to either add a test or throw some sort of error? Might be easy for someone to overlook this last part of the kdoc.